chart/templates/configmap.yaml

```diff
@@ -84,6 +84,22 @@ data:
         {{ $key }}: {{ $val | quote }}
         {{- end }}
       {{- end }}
+      tolerations:
```

Comment:
Could we wrap this key in an `{{- if .Values.injector.tolerations }}` `{{- end }}` block, the way that `annotations` and `labels` are?

Reply:
Good call updated!